index.js

```diff
@@ -85,22 +86,32 @@ const queryClient = new QueryClient( {
 // TODO: remove paper themes from app in favor of Tailwind
 const theme = {
   ...DefaultTheme,
-  roundness: 2,
   version: 3,
   colors: {
     ...DefaultTheme.colors,
-    primary: colors.inatGreen,
-    secondary: colors.secondary,
-    tertiary: colors.tertiary,
-    surface: colors.white
+    primary: colors.darkGray,
+    onPrimary: colors.white,
+    secondary: colors.focusGreen,
+    onSecondary: colors.white,
+    background: colors.white,
+    error: colors.warningRed,
+    onError: colors.white
   }
 };
 
+// eslint-disable-next-line react/jsx-props-no-spreading
+const renderCustomIcon = props => <INatIcon {...props} />;
+
 const AppWithProviders = ( ) => (
   <QueryClientProvider client={queryClient}>
     <RealmProvider>
       <SafeAreaProvider>
-        <PaperProvider theme={theme}>
+        <PaperProvider
+          settings={{
+            icon: renderCustomIcon
+          }}
+          theme={theme}
+        >
```

Comment:
Ignore if you think this is out of scope, but perhaps we could move all this RN Paper setup into a custom `<InatPaperProvider>` component and file?